db/migrations/02_peeps_table_setup.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE peeps(id SERIAL PRIMARY KEY, user_id int REFERENCES users(id), content varchar(140), date_posted date, time_posted time);
```

Comment:
Date_posted and Time_posted can be merged into one field. Ruby has DateTime or just using a Time object would still work.